guide/voice/seeking.md

```diff
@@ -0,0 +1,79 @@
+# Seek
+
+Currently, there's no way to seek audio resources with raw discord.js such as:
+
+```js
+const { createAudioResource } = require('@discordjs/voice');
+
+const resource = createAudioResource();
+
+resource.seek(1000); // This will be an error!
+```
+
+But some packages provide seek options with these solutions.
+
+## Cheat sheet
+
+Methods
+
+1. Play audio
+2. Create seeked resource
+3. Play the seeked audio resource
+
+Yo, this should be easy, right?
+
+### Creation
+
+First, to seek resource, you have to create new function like this:
+
+```js
+// Require necessary package
+const prism = require('prism-media');
+
+
+// stream -> should be audio stream
+// seek -> should be int that within duration of the stream
```

Comment:
It might be better to explain these as text after this example

Reply:
k